libsolidity/ast/Types.cpp

```diff
@@ -566,12 +575,30 @@ BoolResult IntegerType::isImplicitlyConvertibleTo(Type const& _convertTo) const
 
 BoolResult IntegerType::isExplicitlyConvertibleTo(Type const& _convertTo) const
 {
-	return _convertTo.category() == category() ||
```

Comment:
What about implicit conversion?

Reply:
It's redundant as it's already included in if-branch where `convertTo` is an `IntegerType`.

Reply:
I added the check for implicit conversion.